src/tablecellproperties/commands/tablecellpropertycommand.js

```diff
@@ -98,4 +95,28 @@ export default class TableCellPropertyCommand extends Command {
 	_getValueToSet( value ) {
 		return value;
 	}
+
+	/**
+	 * Returns a single value for selected table cells. If all cells have the same value set it returns that value or undefined otherwise.
+	 *
+	 * @param {Array.<module:engine/model/element~Element>} tableCell
+	 * @returns {*}
+	 * @private
+	 */
+	_getSingleValue( tableCell ) {
+		const firstCellValue = this._getAttribute( tableCell[ 0 ] );
+
+		const everyCellHasAttribute = tableCell.every( tableCell => this._getAttribute( tableCell ) === firstCellValue );
+
+		return everyCellHasAttribute ? firstCellValue : undefined;
```

Comment:
So the idea is that somewhere in https://github.com/ckeditor/ckeditor5/issues/6320 this helper will return `'multiple'`?

Reply:
The idea is to have another property for multiple values (Command#values). But I didn't give a second thought to the 6320.

So this method might not change for the `multiple` display. I think that we should expose multiple values (or some getter to check for that case) in the command and implement how to show this in the UI. Command#values _might_ be usable for other features and showing "multiple" in the form looks like a narrow case of the UI.